pulsar-broker/src/test/java/org/apache/pulsar/broker/transaction/TransactionTest.java

```diff
@@ -139,4 +141,22 @@ protected void cleanup() throws Exception {
                 .enableBatchIndexAcknowledgment(true)
                 .subscribe();
     }
+
+    @Test
+    public void testGetTxnID() throws Exception {
+        Awaitility.await().atMost(4, TimeUnit.SECONDS).until(()->{
```

Comment:
Why do we need Awaitility here? I think the transaction can be created directly.

Reply:
TransactionCoodinator may not be ready

Reply:
Ok, I see. Could you please help add a comment? This will help others to understand.
Related to this fix: https://github.com/apache/pulsar/pull/11357